ja_jp.lang

```diff
@@ -239,23 +283,252 @@ gui.xaero_sort_unsorted=なし
 gui.xaero_sort_name=名前
 gui.xaero_sort_symbol=頭文字
 gui.xaero_sort_distance=距離
+gui.xaero_box_cave_maps=地下にある洞窟や建物の構造を表示するために、表面よりも深いブロックを操作するマッピングモードです。 このサイズを超える固体ブロックの正方形をプレイヤーの頭上で検知したときに有効化されます。
+gui.xaero_box_cave_maps2=地下にある洞窟や建物の構造を表示するために、表面よりも深いブロックを操作するマッピングモードです。 このサイズを超える固体ブロックの正方形をプレイヤーの頭上で検知したときに有効化されます。 この固体ブロックは同じYにある必要はありません。
```

Comment:
Doesn't seem correct. Translates back as "building structures" what was originally "interiors of buildings". Please confirm if this makes sense.
Also translates back as "Triggered when a square of solid blocks larger than this size is detected above the player's head. This solid block does not have to be in the same Y." Nothing needs to be "larger", it is looking for the exact size. 
And the second sentence about Y level seems weird too. Make sure it means "the solid blocks don't have to be all on the same Y level to form the square".